pkg/logs/input/container/launcher.go

```diff
@@ -24,26 +24,18 @@ import (
 // the launcher will attempt to initialize a kubernetes launcher which will detect and tail all the logs files localized
```

Comment:
this needs to be updated ` // When a docker launcher can not be initialized properly and when the log collection is enabled for all containers,`

Reply:
as well as the header of this method